tests/test_cli.py

```
@@ -0,0 +1,20 @@
+import shutil
+import subprocess
+
+import pytest
+
+
+@pytest.mark.parametrize("cmd", ["data", "archive", "fs", "checkhealth"])
+def test_cli_commands(cmd: str):
+    subprocess.run(
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)